gensim/models/translation_matrix.py

```diff
@@ -208,8 +208,8 @@ def translate(self, source_words=None, topn=5, additional=None, source_lang_vec=
         if source_words is None:
             raise RuntimeError("The words to be translated must be provided.")
 
-        if not isinstance(source_words, [string_types, list]):
-            raise ValueError("The word to be translated must be string type or a list of string.")
+        if not isinstance(source_words, (string_types, list, tuple)):
```

Comment:
This looks rather unpythonic; why is such test needed?

Python uses duck-typing.